prow/plugins/approve/approvers/owners.go

```diff
@@ -607,7 +607,7 @@ Approval requirements bypassed by manually added approval.
 This pull-request has been approved by:{{range $index, $approval := .ap.ListApprovals}}{{if $index}}, {{else}} {{end}}{{$approval}}{{end}}
 
 {{- if (and (not .ap.AreFilesApproved) (not (call .ap.ManuallyApproved))) }}
-To fully approve this pull request, please assign additional approvers.
+To fully approve this pull request, see our [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process).
```

Comment:
Wdyt about keeping `please assign additional approvers` and appending a line about the pull request process? The process linked in the doc is pretty detailed and I'm not sure everyone will read that.

`please assign additional approvers` gives the tldr and I think it be useful to keep it here.

Reply:
Is it intended that, in most cases, a new user should just /assign whoever the robot suggests? I had thought the answer was "no", in which case, a link to the documentation to determine what to do next is necessary.

If we want people to, usually, just do as the robot says, then perhaps the following:

> To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign nikhita
> You can assign the PR to them by writing /assign @nikhita in a comment when ready

Reply:
We want to encourage people to use /assign, yes. People are too timid about using it.

Reply:
@chases2 Your suggestion looks good.  Just note how we quote `@name` so that it doesn't register as a notification

Reply:
@chases2 Sounds awesome! :100:

Reply:
Fantastic! Thank you all so much for the feedback. The change has been made.